ParseTree/ParseTree/Operand.cs

```diff
@@ -0,0 +1,22 @@
+namespace ParseTree;
+
+/// <summary>
+/// Class realizes operand node of Parse tree.
+/// </summary>
+/// <param name="operand">Integer number.</param>
+public class Operand(int operand) : IParseTreeNode
+{
+    private int operand = operand;
+
+    /// <inheritdoc/>
+    public double Evaluate()
+    {
+        return this.operand;
+    }
```

Comment:
```suggestion
    public double Evaluate()
        => this.operand;
```
И Print можно упростить